pkg/k8shandler/collection.go

```diff
@@ -83,48 +83,50 @@ func CreateOrUpdateCollection(cluster *logging.ClusterLogging) (err error) {
 		if retryErr != nil {
 			return fmt.Errorf("Failed to update Cluster Logging Fluentd status: %v", retryErr)
 		}
-	} else {
-		removeFluentd(cluster)
```

Comment:
this needs to stay here, this is in the case where a user removes the fluentd spec from the CR after it has been initially installed

Reply:
reverted but needs to be pushed